pilz_teleoperation/README.md

```diff
@@ -20,20 +20,14 @@ Use following Syntax:
     ```
     MovementBindings:
       'w': <Twist> (e.g. {"linear": {"x": 1.0}})
-      '<CURSES_KEY_NAME>': <Twist> (e.g. KEY_LEFT)
     SettingBindings:
-      '+': <SettingName> (e.g. USE_XZ_PLANE)
+      '+': <SettingName> (e.g. INCREASE_LINEAR_VELOCITY)
     Description: |-
       Displayed Text e.g. to show current bindings
       With the '|-' the text can contain multiple rows
     ```
     
-    See [SetTeleopSettings.srv](./srv/SetTeleopSettings.srv) for available settings.
```

Comment:
why is this removed?

Reply:
point can stay id say, or remove them in the other lines as well

Reply:
Bullet points are all contained in readme, see https://github.com/PilzDE/pilz_teach/blob/usability_test_additions/pilz_teleoperation/README.md